### PR TITLE
Handle wallet errors

### PR DESCRIPTION
**Pull request by gabaldon**
This PR unifies the way to handle errors covering all wallet calls.

### CODE REVIEW

File: src/components/RadonStage.vue [hunk 1]
```diff
@@ -18,7 +18,6 @@
     />
     <p class="error" v-show="error.aggregate">There is an error in the aggregate stage</p>
   </div>
-
```
Comment (Tommytrg) [inline src/components/RadonStage.vue]: Discard this change

File: src/components/steps/WalletDisclaimer.vue [hunk 2]
```diff
@@ -42,7 +42,6 @@ export default {
       this.$router.push('/ftu/seed-type-selection')
     },
     nextStep() {
-      this.$store.dispatch('createMnemonics')
```
Comment (Tommytrg) [inline src/components/steps/WalletDisclaimer.vue]: Why do you delete the `createMnemonics` dispatch?

File: src/components/steps/WalletEncryptionPassword.vue [hunk 3]
```diff
@@ -45,47 +47,54 @@ export default {
...
-      }
-
-      if (this.password !== this.repeatPassword) {
-        this.error = 'Passwords must match'
-        return false
-      }
-
-      return true
+      this.$store.commit('validatePassword', {
+        password1: this.password,
```
Comment (Tommytrg) [inline src/components/steps/WalletEncryptionPassword.vue]: can you rename it as `password` and `repeatedPassword`?

File: src/components/steps/WalletEncryptionPassword.vue [hunk 4]
```diff
@@ -45,47 +47,54 @@ export default {
...
     },
+    clearError(errorName) {
+      this.$store.commit('clearError', { error: errorName })
+    },
   },
   computed: {
     ...mapState({
       mnemonics: state => state.wallet.mnemonics,
+      createWalletError: state => state.wallet.errors.createWallet,
+      createValidPasswordError: state => {
```
Comment (Tommytrg) [inline src/components/steps/WalletEncryptionPassword.vue]: Can you remove brackets here?

File: src/store/wallet.js [hunk 5]
```diff
@@ -41,10 +49,20 @@ export default {
...
+        this.commit('setError', {
+          name: 'network',
+          error: 'The wallet or the node is not running properly',
+          message: 'connection error',
+        })
+      }
     },
     setDataRequestResult(state, result) {
       Object.assign(state, { radRequestResult: result })
+      console.log('radRequestResult', state.radRequestResult)
```
Comment (Tommytrg) [inline src/store/wallet.js]: Remove console.log

File: src/components/steps/WalletDisclaimer.vue (hunk 2 above)
Comment (gabaldon) [reply src/components/steps/WalletDisclaimer.vue]: I add createMnemonics before the component is created cause mnemonics were not shown when reloading the page.

**Pull request merged**